ir_beacon/CMakeLists.txt

```diff
@@ -66,10 +69,10 @@ find_package(catkin REQUIRED COMPONENTS
 # )
 
 ## Generate added messages and services with any dependencies listed here
-# generate_messages(
-#   DEPENDENCIES
-#   std_msgs  # Or other packages containing msgs
-# )
+ generate_messages(
+   DEPENDENCIES
+   std_msgs  # Or other packages containing msgs
```

Comment:
Is std_msgs necessary? The message looks like it's only using primatives.

Reply:
true, removed dependency on std_msgs af4ae88